src/server/course/course.service.ts

```diff
@@ -75,10 +76,13 @@ export class CourseService {
       .addSelect('c."sameAsId"', 'sameAsId')
       .addSelect("CONCAT_WS(' ', p.prefix, p.number)", 'sameAs')
       .addSelect('c.private', 'private')
+      .addSelect('c."numberInteger"', 'numberInteger')
+      .addSelect('c."numberAlphabetical"', 'numberAlphabetical')
       .leftJoinAndSelect(Area, 'a', 'c."areaId" = a.id')
       .leftJoinAndSelect(Course, 'p', 'c."sameAsId" = p.id')
       .orderBy('a.name', 'ASC')
-      .addOrderBy('"catalogNumber"', 'ASC')
+      .addOrderBy('"numberInteger"', 'ASC')
```

Comment:
This is good, but I think we need `.addOrderBy('prefix')` before the `numberInteger` to address areas that have courses under multiple prefixes:

![2023-01-11_135404](https://user-images.githubusercontent.com/6124793/211894163-62c6215e-b398-49e0-88dc-d8426cf8846e.png)

Reply:
Ah, ok, hadn't thought about that, thanks! I updated the ordering in commit [d69155b](https://github.com/seas-computing/course-planner/pull/621/commits/d69155b337fb61a5cc5d37335cbe559bec40b830) to include ordering by the prefix.